levelkdev/swapr-dapp
# Feature/914 toast message

# Summary

Doesn't entirely fix #914 as written.
What it does is display in the toast the number in the etherscan, or other relevant transaction logger, with the same precision as the computed output.
Some disparities might still appear.

![Screenshot from 2022-06-03 10-16-10](https://user-images.githubusercontent.com/36545016/171826661-9480cb94-65c9-42f8-a3fe-894cf6e30535.png)
![Screenshot from 2022-06-03 10-16-24](https://user-images.githubusercontent.com/36545016/171826703-f39313d6-89c0-4012-bcdb-3d27c13503af.png)

  # To Test

(same as in the raised issue)

1. Go to Swap page
2. Switch to Rinkeby network
3. Swap 0.1 ETH for USDC
4. Check if transaction value in toast message is correct to balance in wallet

(I didn't check the balance in the wallet but the transaction value might also be a good reference)

I'd like to thank Velu's collaboration on this PR's development.

~~EDIT: I noticed the failed checks, I'll address them before merging the PR. Feel free to skip review for now if it's something that makes you uncomfortable.~~  
Fixed this ☝🏼
